### PR TITLE
Force utf8 encoding in setup

### PR DESCRIPTION
**Pull request by jrapin**
#24

### CODE REVIEW

**Comment by gwenzek** [comment]
I'd prefer to just make the encoding explicit in the setup.py.
It's to easy to add a non Ascii char in the Readme, and your name shouldn't need to be butchered like this.

**Comment by jrapin** [comment]
> I'd prefer to just make the encoding explicit in the setup.py.
> It's to easy to add a non Ascii char in the Readme, and your name shouldn't need to be butchered like this.

yes I read that last part after making the PR and then had to leave, I'll fix this

File: setup.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     description="Python 3.6+ toolbox for submitting jobs to Slurm",
     author="Facebook AI Research",
     python_requires=">=3.6",
-    long_description=Path("README.md").read_text(),
+    long_description=Path("README.md").read_text(encoding="utf-8"),
```
Comment (jrapin) [inline setup.py:30]: @gwenzek that should be enough right? not too sure how to test

Comment (gwenzek) [reply setup.py:30]: yes this has been tested in #24

**Pull request merged**